README.md

```diff
@@ -31,10 +31,10 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: 'Transition Jira Ticket to Deployed Status'
-        uses: im-open/transition-jira-tasks-by-query@v1.0.0
+        uses: im-open/transition-jira-tasks-by-query@v1.1.0
         with:
           domain-name: 'jira.com'
-          jql-query: 'issuekey=PROJ-12345'
+          jql-query: 'issuekey=PROJ-12345' or "filter='My Filter Name'"
```

Comment:
I think you need quotes around the whole statement. So something like `"'issuekey=PROJ-12345' or \"filter='My Filter Name'\""`

Reply:
You don't.  This works.  I have tried it.

Reply:
Well I think you are misunderstanding what this is saying... Its not one query.  I am giving an example of one "or" the other.

Reply:
Yes I was misunderstanding. Removing the 'or' and making a commented out second jql-query makes things clearer.

So now I have a new question: what's the purpose of the second query? Was it to try to make it clear that you could have any arbitrary query and not just `issuekey=<value>`? If so, I think this may add more confusion. Something like `jql-query: 'issuekey=PROJ-12345' # This can be any JQL query` might be more clear. Or maybe
```yaml
jql-query: 'issuekey=PROJ-12345'
# jql-query: "filter='My Filter Name'"
# jql-query: 'issuekey=KEY-123 OR project=PROJ'
# etc.
```